api/lang/runner.go

```diff
@@ -31,7 +31,7 @@ var lRunner langRunner
 func startRunner() error {
 	var err = connectToRunner()
 	if err != nil {
-		return fmt.Errorf("unable to connect to runner : %s", err.Error())
+		return fmt.Errorf("Gauge could not connect to the runner. Check the logs for more info")
```

Comment:
```suggestion
		return fmt.Errorf("Gauge could not run the tests. Check the logs for more info")
```

Reply:
This error is not meant for Execution. It's while starting the LSP server. Not sure if it should say something about Execution, could be confusing.

Reply:
`the runner` is a bit confusing. Can this be
`Gauge could not initialise. Check the logs for more info`?

Reply:
👍